scss/components/table.scss

```diff
@@ -82,20 +83,19 @@ $block: #{$fd-namespace}-table;
             }
         }
     }
-    tbody {
-        tr {
+    &__body {
+        &-row {
             border-style: solid;
             border-width: $fd-table-border-width;
             @include fd-var-color("border-color", $fd-table-border-color, --fd-table-border-color);
             border-left: none;
             border-right: none;
         }
-        td {
+        &-col {
             word-break: break-word;
         }
     }
-    td,
-    th {
+    &__col {
```

Comment:
fd-table__col will cover both td and th 👍

Reply:
I know, I'm speaking more about the name. The selector `td, th` was previously "any cell in a table" whereas `fd-table__col` implies it's just for columns...

I'm just always trying to wear the maintainability/are-we-going-to-understand-this-in-3-months hat 🎩

Reply:
Maybe `fd-table__cell` might be a better fit here?

Reply:
@greg-a-smith thoughts? I believe it might be best then to have this split up into `fd-table__header-cell` and `fd-table__col` instead of just `fd-table__cell` for ease on labeling purposes? Otherwise I will need to label all <td> tags with `fd-table__col` and `fd-table-cell`. 

With them split up I can label all `<th>` with `fd-table__header-cell` and keep all <td> as `fd-table__col`

Reply:
@hertweckhr1 I like @markpalfreeman's suggestion.  He's right that `fd-table__cell` is a better replacement for `td` and `th` than `fd-table__col`.

Reply:
coolio changing it to fd-table__cell on Mark and Jenna's good thinkings 👍